app/views/playerControlsExtended.js

```diff
@@ -104,9 +104,16 @@ const ControlsExtended = Mn.ItemView.extend({
     changeVolWheel(e) {
         const wheelStep = 0.05;
         let volume = application.appState.get('currentVolume', volume);
-        if (e.originalEvent.wheelDelta > 0 ) {
-            volume += wheelStep;
+        let wheeldelta;
+        if (/firefox/i.test(window.navigator.userAgent)) {
```

Comment:
May this [doc](https://www.h3xed.com/programming/javascript-mouse-scroll-wheel-events-in-firefox-and-chrome) can help ?